google-pub/sample.json

```diff
@@ -0,0 +1,51 @@
+{
+    "name": "Google MQTT Publish",
+    "category": "Cloud",
+    "tag": "cloud",
+    "date": "2018-2-21",
+    "description": "An application based on Google* Cloud IoT Core for publishing a single message. See the README.md file in the GitHub repo for this project before continuing.",
+    "platform": {
+	"libs":[
+		"Jannson",
+		"libjwt",
+		"OpenSSL",
+		"paho.mqtt.c",
+	],
+	"os": ["ubuntu-desktop-xenial-64bit"]
+    },
+    "service_provider": "Google Cloud IoT Core",
+    "service_provider_uri": "https://cloud.google.com/iot-core/m",
+    "service_provider_description": "A fully managed service to easily and securely connect, manage, and ingest data from globally dispersed devices",
+	"additional_fields":[
```

Comment:
Please add the readme link here as well:
"sample_readme_uri": "https://github.com/intel-iot-devkit/iot-devkit-samples/blob/iss2018-update1/google-pub/c/README.md",

Reply:
Ah good catch.  I have added a URL but I think it would change once the merge happens. Will update again.